kubernetes/test-node-base.yaml

```diff
@@ -14,15 +14,27 @@ spec:
       labels:
         host: test-nodeX
     spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 100
+              podAffinityTerm:
+                labelSelector:
+                  matchExpressions:
+                    - key: app
+                      operator: In
+                      values:
+                        - multi-node-test
+                topologyKey: kubernetes.io/hostname
       containers:
       - image: openjdk:11
         command: ["sleep", "infinity"]
         resources:
           requests:
-            memory: "4Gi"
-            cpu: "1800m"
+            memory: "3Gi"
+            cpu: "1400m"
```

Comment:
n2-standard-4 should have 4 vCPU so this shouldn't be the problem, but reducing it a bit anyway. 1400m still means that only 2 pods will fit in one node